CMakeLists.txt

```diff
@@ -16,24 +16,78 @@ project(
     LANGUAGES CXX
 )
 
-#---- C++ standard that we need (at least).
-set(CMAKE_CXX_STANDARD          20   CACHE BOOL "" FORCE)
-set(CMAKE_CXX_STANDARD_REQUIRED True CACHE BOOL "" FORCE)
+#---- Options.
+option(KOKKOSUTILS_ENABLE_TESTS "Enable testing" ON)
+option(KOKKOSUTILS_ENABLE_DOC   "Enable documentation" ON)
 
 #---- Find Kokkos.
-find_package(
-    Kokkos
-    ${KOKKOS_VERSION}
-    REQUIRED
-)
+#
+# Currently, we require the Kokkos::kokkoscore target. Other targets will be added as needed.
+if(NOT TARGET Kokkos::kokkoscore)
+    find_package(
+        Kokkos
+        ${KOKKOS_VERSION}
+        CONFIG
+        REQUIRED
+    )
```

Comment:
Find or file issue about allowing components to be passed.

Reply:
To be done later.

Reply:
For later.